tests/backup/backup_resiliency_test.go

```diff
@@ -1189,3 +1190,231 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			for _, namespace := range appNamespaces {
+				sem <- struct{}{}
+				backupName := fmt.Sprintf("%s-%s", BackupNamePrefix, namespace)
+				backupNames = append(backupNames, backupName)
+				wg.Add(1)
+				go func(backupName string, namespace string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					defer func() { <-sem }()
+					// Here we are using destination cluster as application cluster
+					_, err = CreateBackupWithoutCheck(backupName, destinationClusterName, bkpLocationName, backupLocationUID,
+						[]string{namespace}, labelSelectors, orgID, destClusterUid, "", "", "", "", ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Taking backup %s of application- %s", backupName, namespace))
+				}(backupName, namespace)
+			}
+			wg.Wait()
+			log.Infof("The list of backups taken are: %v", backupNames)
+		})
+		Step("Reboot one worker node on application cluster when backup is in progress", func() {
+			log.InfoD("Switching cluster context to application[destination] cluster")
+			SetDestinationKubeConfig()
+			listOfWorkerNodes = node.GetWorkerNodes()
+			err := Inst().N.RebootNode(listOfWorkerNodes[0], node.RebootNodeOpts{
+				Force: true,
+				ConnectionOpts: node.ConnectionOpts{
+					Timeout:         rebootNodeTimeout,
+					TimeBeforeRetry: rebootNodeTimeBeforeRetry,
+				},
+			})
+			dash.VerifySafely(err, nil, fmt.Sprintf("Rebooting worker node %v", listOfWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			dash.VerifyFatal(err, nil, "Switching back context to source cluster")
+
+		})
+		Step("Check if backup is successful after one worker node on application cluster is rebooted", func() {
+			log.InfoD("Check if backup is successful after one worker node on application cluster is rebooted")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, backupName := range backupNames {
+				err := backupSuccessCheck(backupName, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+				dash.VerifySafely(err, nil, "Inspecting backup success for - "+backupName)
+			}
+		})
+		Step("Check if the rebooted node on application cluster is up now", func() {
+			log.InfoD("Check if the rebooted node on application cluster is up now")
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			listOfWorkerNodes = node.GetWorkerNodes()
+			nodeReadyStatus := func() (interface{}, bool, error) {
+				err := Inst().S.IsNodeReady(listOfWorkerNodes[0])
+				if err != nil {
+					return "", true, err
+				}
+				return "", false, nil
+			}
+			_, err := task.DoRetryWithTimeout(nodeReadyStatus, K8sNodeReadyTimeout*time.Minute, K8sNodeRetryInterval*time.Second)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the status of rebooted node %s", listOfWorkerNodes[0]))
```

Comment:
Do no print out the the whole Node instance. Pick and choose only the relevant stuff you want out of it.

Reply:
done